certified-connectors/ShareEffect/apiDefinition.swagger.json

```diff
@@ -66,8 +66,8 @@
                         "parameter": "termsgroup"
                       }
                     },
-                    "value-path": "key-item-output",
-                    "value-title": "key-item-output"
+                    "value-path": "",
+                    "value-title": ""
```

Comment:
@Rens-Braspenning Please remove the empty value properties i.e. "value-path" and "value-title"